src/ListActions/ListActions.js

```diff
@@ -58,202 +45,89 @@ var ListActionActivationSupport = {
 	}
 };
 
-
 /**
-* Fires when the [ListActionsDrawer]{@link module:moonstone/ListActions~ListActionsDrawer} has completed any
-* setup and/or preparation work, e.g., when animating or initial setup. No event-specific
-* information is sent with this event.
-*
-* @event module:moonstone/ListActions~ListActionsDrawer#onComplete
-* @type {Object}
-* @public
-*/
-
-/**
-* {@link module:moonstone/ListActions~ListActionsDrawer} is a [control]{@link module:enyo/Control~Control} used by
+* {@link module:moonstone/ListActions~ListActionsPopup} is a
+* [control]{@link module:moonstone/ContextualPopup~ContextualPopup} used by
 * {@link module:moonstone/ListActions~ListActions} to house a menu of selectable options.
 *
-* @class ListActionsDrawer
-* @extends module:enyo/Control~Control
+* @class ListActionsPopup
+* @extends module:moonstone/ContextualPopup~ContextualPopup
 * @ui
-* @public
+* @private
 */
-var ListActionsDrawer = kind(
-	/** @lends module:moonstone/ListActions~ListActionsDrawer */ {
+var ListActionsPopup = ContextualPopup.kind(
+	/** @lends module:moonstone/ListActions~ListActionsPopup */ {
 
 	/**
 	* @private
 	*/
-	name: 'moon.ListActionsDrawer',
+	classes: 'moon-list-actions-popup',
 
 	/**
-	* @private
+	* @see moonstone/ContextualPopup~ContextualPopup#spotlightModal
 	*/
-	kind: Control,
-
-	/**
-	* @private
-	* @lends module:moonstone/ListActions~ListActionsDrawer.prototype
-	*/
-	published: {
-
-		/**
-		* If `true`, the drawer will be in its opened state; otherwise, it will be closed.
-		*
-		* @type {Boolean}
-		* @default false
-		* @public
-		*/
-		open: false
-	},
-
-	/**
-	* @private
-	*/
-	handlers: {
-		onPanelOnscreen: 'hideCloseButton'  // Re-evaluate whether ListActions should be open and if the close X should show or not.
-	},
-
-	/**
-	* @private
-	*/
-	classes: 'moon-list-actions-drawer',
-
-	/**
-	* @private
-	*/
-	components: [
-		{name: 'client', kind: Control, classes: 'moon-list-actions-drawer-client moon-neutral', ontransitionend: 'handleTransitionEnd'}
-	],
+	spotlightModal: true,
 
 	/**
+	* Adjust popup direction, anchor to the edge of screen if it goes over, and adjust arrow
+	* positions.
+	*
 	* @private
-	*/
-	events: {
-		onComplete: '',
-		onCustomizeCloseButton: ''
-	},
+	* @override
+	*/
+	alterDirection: function () {
+		if (this.showing) {
+			var clientRect = this.getBoundingRect(this.node),
+				viewPortWidth = dom.getWindowWidth(),
+				offsetWidth = (clientRect.width - this.activatorOffset.width) / 2,
+				popupMargin = ri.scale(24),
+				iconButtonWidth = this.activatorOffset.width + popupMargin,
+				bounds = {top: null, left: null},
+				c;
+
+			this.resetDirection();
+			this.addClass('below');
```

Comment:
Any reason we can't include this in the classes for the kind since we're always 'below'?